tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -39,8 +37,8 @@ const expectTargets = async (targets) => {
 };
 
 const openTargetDetails = async (targetID) => {
-  await element(by.css(`#target-aggregates-list li[data-record-id=${targetID}] a`)).click();
-  await helper.waitElementToPresentNative(element(by.css('.target-detail.card h2')));
+  await $(`#target-aggregates-list li[data-record-id=${targetID}] a`).click();
```

Comment:
Can you put the selectors definitions into the corresponding page-object file please?